src/schemas/validation/schema.yaml

```diff
@@ -8,6 +8,9 @@ properties:
   openapi:
     type: string
     pattern: '^3\.1\.\d+(-.+)?$'
+  $self:
+    type: string
+    format: uri-reference
```

Comment:
we could add `regex: '^[^#]+$'` here, but I think this would be better done as a definition so the other uri-references can also benefit (will todo as a separate item).

Reply:
Added to this PR.